backend/package.json~

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
Please remove this file. For the future, `git add .` and `git diff --cached` is your friend, prior to `git commit`.